README.Rmd

```diff
@@ -40,100 +40,30 @@ admiralci_homepage <- "https://pharmaverse.github.io/admiralci"
 <!-- The purpose of this is to enable dynamic links using dyn_link function above to access devel/main admiral homepage respectively -->
 <!-- To test this in your feature branch use code: rmarkdown::render("README.Rmd", output_format ="md_document") -->
 
-# admiral package extension Template <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
+# pharmaverseadam <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
 
 <!-- badges: start -->
 [<img src="http://pharmaverse.org/shields/admiral.svg">](https://pharmaverse.org)
 [![CRAN status](https://www.r-pkg.org/badges/version/admiral)](https://CRAN.R-project.org/package=admiral)
 [![Test Coverage](https://raw.githubusercontent.com/pharmaverse/admiraltemplate/badges/main/test-coverage.svg)](https://github.com/pharmaverse/admiraltemplate/actions/workflows/code-coverage.yml)
 <!-- badges: end -->
 
-## Table of Contents
+Test data (ADaM) for the pharmaverse family of packages
 
-- `r link("Extension Expectations", "#extension-expectations")`
-- `r link("What is included in the template", "#what-is-included-in-the-template")`
-- `r link("Package Extension Guidance", "#package-extension-guidance")`
-- `r link("Quick Start Guide for Template", "#quick-start-guide-for-template")`
+# Purpose
 
-## Extension Expectations 
+To provide a one-stop-shop for ADaM test data in the pharmaverse family of packages. This includes test ADaM data created from {admiral} templates. 
 
-To be included as an `{admiral}` package extension we expect developers to adhere to the following standards:
+There is an integration test that runs {admiral} templates and compares them with the stored version. If differences are found, it then triggers an action: either accepting the difference and storing the new ADaM, or rejecting it and identifying the cause of the differences in the templates.
```

Comment:
Maybe this sentence could be a little clearer. I would suggest going from: 

_There is an integration test that runs {admiral} templates and compares them with the stored version. If differences are found, it then triggers an action: either accepting the difference and storing the new ADaM, or rejecting it and identifying the cause of the differences in the templates._

to:

_The ADaM contents of this package is automatically populated by a CICD action that executes `{admiral}` templates and saves the resulting dataset here. The ADaM datasets in `{pharmaversesdtm}` are updated any time the templates are updated in `{admiral}`_